src/app/components/abstract/abstract-form-parent-base/abstract-form-parent-base.component.ts

```diff
@@ -25,7 +25,23 @@ export abstract class AbstractFormParentBaseComponent {
     }
   }
 
+  /**
+   * Navigates to the specified route using the Angular router.
+   *
+   * @param route - The route to navigate to.
+   */
   protected routerNavigate(route: string): void {
     this.router.navigate([route], { relativeTo: this.activatedRoute.parent });
   }
+
+  /**
+   * Checks if an object has any non-empty values.
+   *
+   * @param form - The object representing form data with key-value pairs.
+   * @returns A boolean indicating whether the object has any non-empty values.
+   */
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  protected hasFormValues(form: { [key: string]: any }): boolean {
```

Comment:
could use { [key: string]: unknown } instead of any so here you don't have to silence linter

Reply:
We do this in other places: [abstract-form-base.component.ts](https://github.com/hmcts/opal-frontend/blob/c2f5aec5cc409c1098c669e7cf4ea447e29e0f58/src/app/components/abstract/abstract-form-base/abstract-form-base.component.ts#L439)

Gave it a go changing it to unknown but gives me an error when trying to use it within fines-mac-account-notes.component.ts